CHANGELOG.md

```diff
@@ -1,9 +1,53 @@
-# dbt_fivetran_log version.version
+# dbt_fivetran_log v1.11.0
+[PR #144](https://github.com/fivetran/dbt_fivetran_log/pull/144) includes the following updates:
+
+## Breaking Changes
+> A `--full-refresh` is **required** after upgrading to prevent errors caused by naming changes.
+
+- Source Table Transition:
+  - In Q1 2025, the source table `CONNECTOR` was replaced with a new table, `CONNECTION`. Historical data remains in `CONNECTOR` and is not migrated to `CONNECTION`.
```

Comment:
> Historical data remains in `CONNECTOR` and is not migrated to `CONNECTION`.

do you know if this is the case even if a customer were to do a historical resync of the connection?